app/questionnaire/router.py

```diff
@@ -139,13 +146,13 @@ def get_first_incomplete_location_in_survey_url(self):
 
         return self.get_last_location_in_survey().url()
 
-    def get_section_resume_url(self, routing_path):
+    def get_section_resume_url(self, routing_path, resume=True):
         section_key = (routing_path.section_id, routing_path.list_item_id)
 
         if section_key in self._progress_store:
             location = self._get_first_incomplete_location_in_section(routing_path)
             if location:
-                return location.url(resume=True)
+                return location.url(resume=resume) if resume else location.url()
```

Comment:
I don't think these changes are needed now that the extra call to this method has been removed.

Reply:
Removed, thanks👍